libr/core/p/core_java.c

```diff
@@ -429,12 +429,12 @@ static int r_cmd_java_handle_summary_info(RCore *core, const char *cmd) {
 	}
 
 	r_cons_printf ("Summary for %s:\n", obj->file);
-	r_cons_printf ("  Size 0x%" PFMT64x ":\n", obj->size);
-	r_cons_printf ("  Constants  size: 0x%" PFMT64x " count: %d:\n", obj->cp_size, obj->cp_count);
-	r_cons_printf ("  Methods    size: 0x%" PFMT64x " count: %d:\n", obj->methods_size, obj->methods_count);
-	r_cons_printf ("  Fields     size: 0x%" PFMT64x " count: %d:\n", obj->fields_size, obj->fields_count);
-	r_cons_printf ("  Attributes size: 0x%" PFMT64x " count: %d:\n", obj->attrs_size, obj->attrs_count);
-	r_cons_printf ("  Interfaces size: 0x%" PFMT64x " count: %d:\n", obj->interfaces_size, obj->interfaces_count);
+	r_cons_printf ("  Size 0x%x:\n", obj->size);
+	r_cons_printf ("  Constants  size: 0x%x count: %d:\n", obj->cp_size, obj->cp_count);
```

Comment:
if its hexa is probably good to add some padding like 0x%08, but i think this output is there for dexdump compat, so better keep it like this (in other words: ignore this comment)